Scripts/ProjectileMovement.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class ProjectileMovement : MonoBehaviour
+{
+    public Vector3 target;
+    private Rigidbody2D rb;
+    public float force;
+    public AudioClip[] death;
+
+    private void Start()
+    {
+        rb = GetComponent<Rigidbody2D>();
+        target = GameManager.current.playerObject.transform.position;
+        Debug.Log("target is: " + target);
+        Vector3 direction = target - transform.position;
+        rb.velocity = new Vector3(direction.x, direction.y).normalized * force;

```

Comment:
I believe the Rigidbody2D has a method called AddForce, not sure if it's the same